utility/WiFiStream.h

```diff
@@ -124,6 +149,7 @@ class WiFiStream : public Stream
     return result;
   }
 
+#ifndef ESP8266
```

Comment:
WEP is not mentioned [here](https://github.com/esp8266/Arduino/blob/master/doc/libraries.md#wifiesp8266wifi-library).